src/modules/wishlists.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Wishlists reducer
+ * @param {*} state
+ * @param {*} action
+ */
+export default function reducer(state = {}, action) {
```

Comment:
Name the reducers properly as we will have multiple of them, in this case it can be something like wishtlistReducer